README.md

```diff
@@ -4,128 +4,108 @@
 
 [Distributed TSDB](https://docs.m3db.io/m3db/) and [Query Engine](https://docs.m3db.io/how_to/query/), [Prometheus Sidecar](https://docs.m3db.io/integrations/prometheus/), [Metrics Aggregator](https://docs.m3db.io/overview/components/#m3-aggregator), and more such as [Graphite storage and query engine](https://docs.m3db.io/integrations/graphite/).
 
-More information:
+## Table of Contents:
+- [Install](#install)
+- [Dependencies](#dependencies)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Community meetings](#community-meetings)
+- [Office Hours](#office-hours)
+
+## More information
 
 - [Documentation](https://docs.m3db.io/)
 - [Developer Guide: Getting Started](https://github.com/m3db/m3/blob/master/DEVELOPMENT.md)
 - [Slack](http://bit.ly/m3slack)
 - [Forum (Google Group)](https://groups.google.com/forum/#!forum/m3db)
-- [Twitter](https://twitter.com/m3metrics)
-
-## Community meetings
-
-M3 contributors and maintainers have monthly (every four weeks) meetings. Join our M3 meetup group to receive notifications on upcoming meetings: 
-[https://www.meetup.com/M3-Community/](https://www.meetup.com/M3-Community/).
 
-Recordings of past meetups can be found here: [https://vimeo.com/user/120001164/folder/2290331](https://vimeo.com/user/120001164/folder/2290331).
+## Install 
 
-## Office Hours
 
-Members of the M3 team will hold office hours on the third Thursday of every month from 11-1pm EST. In order to join, make sure to sign up for a slot here: [https://calendly.com/chronosphere-intro/m3-community-office-hours](https://calendly.com/chronosphere-intro/m3-community-office-hours).
 
-## Test it out
+### Dependencies 
```

Comment:
Should we mention something about `jq` here?

Reply:
Good catch!